src/kite.js

```diff
@@ -345,26 +322,7 @@ export const Kite = {
       })
     );
 
-    this.disposables.push(
-      vscode.commands.registerCommand("kite.docs-at-cursor", () => {
-        const editor = vscode.window.activeTextEditor;
-
-        if (editor) {
-          const pos = editor.selection.active;
-          const { document } = editor;
-
-          const path = hoverPath(document, pos);
-          KiteAPI.request({ path }).then(resp => {
-            if (resp.statusCode === 200) {
-              vscode.commands.executeCommand("kite.more-position", {
-                position: pos,
-                source: "Command"
-              });
-            }
-          });
-        }
-      })
-    );
+    this.disposables.push(...(new DocsCommands).register())
```

Comment:
What if we instantiated `DocsCommands` outside here? Once more classes implement the `ICommandRegistrant` interface, we could potentially do something like:

```typescript
const commandRegistrants ICommandRegistrant[] = [...];
this.commandRegistrants.map(c => this.disposables.push(...c.register());
```

It's a little overkill now, but it could be helpful to set the intention of how to properly add new commands going forward.